src/components/dashboard/sidebar/SubscriptionSummaryCard.jsx

```diff
@@ -32,7 +32,7 @@ const SubscriptionSummaryCard = ({ subscriptionPlan, className }) => {
       statusBadgeVariant = SUBSCRIPTION_WARNING_BADGE_VARIANT;
     }
     return (
-      <div className="d-flex">
+      <div className="d-flex align-items-end justify-content-between">
```

Comment:
Probably use `align-items-start` here in case the title contents wraps by chance, the badge would stay at the top?